api/lib/application/certification-issue-reports/index.js

```diff
@@ -31,7 +24,6 @@ const register = async function (server) {
         tags: ['api', 'certification-issue-reports'],
         notes: [
           '- **Cette route est restreinte aux utilisateurs qui sont membres de la session du centre de certification**\n',
```

Comment:
On ne peut pas être simplement membre d'une session, si ? 🤔
```suggestion
          '- **Cette route est restreinte aux utilisateurs qui sont membres du centre de certification**\n',
```